Cargo.toml

```diff
@@ -9,8 +9,8 @@ rand_core = "0.5"
 dusk-bytes = "0.1"
 dusk-bls12_381 = {version = "0.6", default-features = false}
 dusk-jubjub = {version = "0.8", default-features = false}
-dusk-poseidon = {version = "0.18", default-features = false}
-dusk-pki = { version = "0.6", default-features = false}
+dusk-poseidon = {version = "0.20", default-features = false}
+dusk-pki = {version = "0.6", default-features = false}
```

Comment:
Shouldn't you use the latest version of `dusk-pki`?

Reply:
The last is `0.6.2` hence `0.6`